web_external/js/MinervaModel.js

```diff
@@ -9,6 +9,14 @@ minerva.models.MinervaModel = girder.models.ItemModel.extend({
         }
     },
 
+    mM: function (minervaMetadata) {
+        if (minervaMetadata) {
```

Comment:
Can we rename this to ```minervaMetadata``` or something that follows suit with the rest of the class?